apps/src/templates/teacherDashboard/SectionAssigner.jsx

```diff
@@ -28,6 +29,22 @@ class SectionAssigner extends Component {
     this.props.selectSection(sectionId);
   };
 
+  // Added this section
```

Comment:
What does this comment mean?

Reply:
It was a note to myself that didn't get deleted :) It is deleted now.